src/command.c

```diff
@@ -204,10 +206,30 @@ handle_command_stop (const struct subcommand *sub,
 	}
 
 	ret = cc_oci_stop (config, state);
+
+        if (state->devices) {
+                /* loading appropriate drivers on the devices */
+		g_debug ("loading drivers on the devices that pass-thru'ed");
```

Comment:
For the message, how about simply `loading drivers for pass-through devices`?

I use vim's `:set spell spelllang=en_gb` which flags this word (as being from another region).